rcgen/src/lib.rs

```diff
@@ -65,7 +67,6 @@ pub use key_pair::{KeyPair, RemoteKeyPair, SubjectPublicKeyInfo};
 use ring_like::digest;
 pub use sign_algo::algo::*;
 pub use sign_algo::SignatureAlgorithm;
-pub use string_types::*;
```

Comment:
Is your intent to make this a semver incompatible change? If so we should add a version bump to [fix the semver check job](https://github.com/rustls/rcgen/actions/runs/14434832706/job/40474197283).

Otherwise, maybe the `use` on L50 should be a `pub use` to continue to allow downstream code to import the string types through the top-level crate namespace?

Reply:
Yeah my idea was to do it now because we want to break the API anyways. I can bump the version, but let's do a release later only.